s/features/notes/components/integrate-notes-components.ts

```diff
@@ -1,6 +1,6 @@
 
 import {XiomeNotes} from "./notes/xiome-notes.js"
-import {XiomeNotesButton} from "./notes-button/xiome-notes-button.js"
+import {XiomeNotesIndicator} from "./notes-button/xiome-notes-button.js"
```

Comment:
- rename the `notes-button/` directory to `notes-indicator/`
- rename `xiome-notes-button.ts` to `xiome-notes-indicator.ts`